client/src/components/Event/Event.css

```diff
@@ -6,10 +6,19 @@
     font-family: 'Raleway';
     color: #EE2A7B;
     font-size: 8vh;
-    width: auto;
-    height: auto;
     opacity: 1;
     margin-bottom: 5vh;
     margin-top: 5vh;
-    margin-left: 40vw;
-}
\ No newline at end of file
+    margin-right: 10px;
```

Comment:
you should use alignment manipulators rather than left/right margins. The alignment should be center at all times.

Reply:
Done.